README.md

```diff
@@ -90,7 +90,16 @@ git clone git@github.com:Sphereon-Opensource/OID4VC-demo.git
 We maintain Docker a setup for building and testing in directory `docker/compose/build`.
 We have created a script to install and patch the agent configurations for you required to build and run the Docker
 containers.
+
```

Comment:
This whole section can go out.
![image](https://github.com/Sphereon-Opensource/OID4VC-demo/assets/26876074/b5fda796-3a9c-4153-ac74-8c9c66b22199)
We don't need jq we don't need choco and a clean Windows machine does not execute .sh scripts at all.

install-configs.sh is part of this block
![image](https://github.com/Sphereon-Opensource/OID4VC-demo/assets/26876074/d2819a5c-6f76-4d5b-9e91-de6b4e08e894)
Which should probably be split up to have this section in between
![image](https://github.com/Sphereon-Opensource/OID4VC-demo/assets/26876074/9ec2c4ad-bbab-4d33-b731-788ca51887ae)